coordinator/gscoordinator/coordinator.py

```diff
@@ -662,13 +691,15 @@ def _RunStep(self, request_iterator, context):
     )(_RunStep)
 
     def _maybe_compile_app(self, op):
-        app_sig = get_app_sha256(op.attr)
+        app_sig = get_app_sha256(op.attr, self._java_class_path)
```

Comment:
Sees the path is a constant and `app_sha256`doesn't need it

Reply:
`get_app_sha256` calls `_codegen_app_info`, the latter defined in `utils.py` and can not get java_class_path, the member field of coordinator. We may need to pass this value.